src/server/game/BattlePets/BattlePetMgr.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2008-2015 TrinityCore <http://www.trinitycore.org/>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef BattlePetMgr_h__
+#define BattlePetMgr_h__
+
+#include "DB2Stores.h"
+#include "BattlePetPackets.h"
+
+#define MAX_PET_BATTLE_SLOTS 3
+#define MAX_BATTLE_PETS_PER_SPECIES 3
+#define BATTLE_PET_CAGE_ITEM_ID 82800
+
+// TODO: fix values in this enum
+enum BattlePetError
+{
+    BATTLEPETRESULT_CANT_HAVE_MORE_PETS_OF_THAT_TYPE = 8,
+
+    // wrong order
+    BATTLEPETRESULT_TOO_HIGH_LEVEL_TO_UNCAGE,
```

Comment:
Shauren discovered the client enum in 6.1.2, but there were only names without the readable values